### ISSUE TITLE
Copy/paste from osx buffer to helm

### ISSUE THREAD

**Issue by idoo** [hlissner/doom-emacs#906]
Sometimes it would be nice to paste something from osx buffer to helm. For example when you're trying to find some code or file by part of the name that you copied from console

But for now, with `mac` feature enabled it's doesn't work (however from edit buffer copy/paste works well)

Any idea how to fix it?

**Comment by hlissner** [comment]
Not sure I understand. You can paste your clipboard to ivy/helm with <kbd>cmd</kbd><kbd>v</kbd> (or the emacs default, <kbd>C-y</kbd>). I can do this from anywhere where you can copy something to  your clipboard. Does this not work on MacOS?

**Comment by idoo** [comment]
sometimes it works, sometimes not — for example, if you'll press `SPC + SPC` for search could you paste filename for example to perform the search?

**Comment by idoo** [comment]
@hlissner do you have any ideas how to fix it?

**Comment by hlissner** [comment]
Sorry for the late response. I still don't quite understand the problem. In `SPC SPC`, pasting appears to work fine and consistently. Are you using helm or ivy?

**Comment by idoo** [comment]
[screencast 2018-10-15 14-29-52.mp4.zip](https://github.com/hlissner/doom-emacs/files/2478362/screencast.2018-10-15.14-29-52.mp4.zip)
please take a look to attached video

I've copied any text from browser and tried to paste it to emacs with CMD+V — as you can see there is no luck

**Comment by idoo** [comment]
@hlissner does it clear for you?

**Comment by Bazai** [comment]
Same problem here. I'm able to paste with `C-y`, but `Cmd-V` doesn't work. Like in @idoo screencast.

**Comment by idoo** [comment]
yes, I can confirm that `C-y` works well, but `Cmd-V` doesn't

**Comment by eduarbo** [comment]
@idoo I have fixed that in #1124 If you don't want to wait just paste the following in your private `+bindings.el`
```elisp
(:when IS-MAC
  (:map general-override-mode-map
    :gi "M-v" #'yank)))
```

**Comment by idoo** [comment]
works well now! thank you <3
